python/ray/rllib/a3c/base_evaluator.py

```diff
@@ -20,6 +20,7 @@ class A3CEvaluator(Evaluator):
 
     Attributes:
         policy: Copy of graph used for policy. Used by sampler and gradients.
+        obs_filter: Observation filter used in environment sampling
```

Comment:
To avoid files called the same name, how about ppo_evaluator (I renamed DQN evaluators in the other PR).